app/Fission/Storage/PostgreSQL.hs

```diff
@@ -1,51 +1,36 @@
 module Fission.Storage.PostgreSQL
-  ( setupTable
-  , connPool
-  , makeTable
+  ( updateDBToLatest
+  , withDBPool
   ) where
 
-import Data.Pool
-import Database.Selda
-import Database.Selda.PostgreSQL
+import qualified Data.ByteString.Char8 as BS8
+import           Data.Pool
 
-import           Fission.Prelude
-import qualified Fission.Storage.Table as Table
-import qualified Fission.Storage.Types as DB
-import           Fission.Internal.Orphanage.Tuple ()
-import           Fission.App (runApp)
-
-setupTable
-  :: ( MonadRIO cfg m
-     , HasLogFunc cfg
-     )
-  => PGConnectInfo
-  -> Table b
-  -> TableName
-  -> m ()
-setupTable pgInfo tbl tblName = do
-  logInfo <| "Creating table `" <> displayShow tblName <> "` in DB"
-  liftIO <| withPostgreSQL pgInfo <| createTable tbl
+import           Database.Esqueleto (SqlBackend, runMigration)
+import           Database.Persist.Postgresql (withPostgresqlPool)
 
-connPool
-  :: ( MonadRIO   cfg m
-     , HasLogFunc cfg
-     )
-  => Int
-  -> Int
-  -> NominalDiffTime
-  -> PGConnectInfo
-  -> m (DB.Pool PG)
-connPool stripeCount connsPerStripe connTTL pgInfo@(PGConnectInfo {..}) = do
-  logDebug <| "Establishing DB pool for " <> displayShow pgDatabase
+import           Fission.Prelude
+import           Fission.Storage.PostgreSQL.ConnectionInfo.Types
 
-  rawPool <- liftIO <| createPool (pgOpen pgInfo) seldaClose stripeCount connTTL connsPerStripe
-  logDebug <| "DB pool stats: " <> displayShow rawPool
+import           Fission.Models
 
-  return <| DB.Pool rawPool
+-- | Automagically update the database schema to reflect the latest schema in Fission.Models
+--
+--   Example:
+--
+--   > runOne (runDB updateDBToLatest)
```

Comment:
☝️ How to apply the changes to a live system

Reply:
The same as would happen in any other system.

_However_ because persistent refuses to _remove_ tables/rows/etc, it actually may just be totally safe to stick right at the top of the server initialization 🤔 Good point

Reply:
Im more on the side of automatically migrating the database on deploy. If things are going wrong at this step for us I think thats more a sign thats something wrong with our steps leading up to deploy rather than our system configuration itself

Sidenote:
> persistent refuses to remove tables/rows/etc
Wait really? Whys that? I understand that might be beneficial to never have to worry about accidentally destroying data. However the trade offs dont see worth it

Reply:
The idea is that `migarteAll` is _so_ automatic that it's easy to accidentally bork critical production server data. If you want to drop a table, rename a column, &c, then you need to move to a more manual process like [`persistent-migration`](https://hackage.haskell.org/package/persistent-migration)

Reply:
> migarteAll is so extremely automatic

I gotcha, that makes sense then. Its not obvious that if you remove a a field from a type that it will remove any related column.

Whats your thoughts on `persistent-migration` then? something we will add?

What are your thoughts on

Reply:
Yeah, exactly. Not something that we need to day, and we can add if/when we start doing a lot of destructive DB changes.